crates/codegen/language/definition/src/compiler/analysis/definitions.rs

```diff
@@ -93,7 +93,6 @@ fn check_precedence_items(analysis: &mut Analysis) {
             for second_op in precedence_expression.operators.iter().skip(1) {
                 if !compare_operators(analysis, first_op, second_op) {
                     analysis.errors.add(name, &Errors::OperatorMismatch);
-                    continue;
```

Comment:
This one I'm still removing

Reply:
Yes, this one is pretty obviously redundant.